.NET/Microsoft.Recognizers.Definitions.Common/Japanese/DateTimeDefinitions.cs

```diff
@@ -47,6 +47,7 @@ public static class DateTimeDefinitions
       public const string DateThisRe = @"这个|这一个|这|这一|本|今";
       public const string DateLastRe = @"上个|上一个|上|上一|去";
       public const string DateNextRe = @"下个|下一个|下|下一|明";

```

Comment:
Ideally we should rename all these regexes to use complete names. No need for such abbreviations, which are also inconsistent with others.